ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +406,193 @@ def test_trustdomain_disable_disables_subdomain(self):
+
+        # add group members
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'b', '--groups', 'a'])
+        assert 'Group name: b' in cmd_output.stdout_text
+        assert 'Member groups: a' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'c', '--groups', 'b'])
+        assert 'Group name: c' in cmd_output.stdout_text
+        assert 'Member groups: b' in cmd_output.stdout_text
+        assert 'Indirect Member groups: a' in cmd_output.stdout_text
+
+        # add user to group 'a'
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'a', '--users', username])
+        assert 'Group name: a' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member of groups: b' in cmd_output.stdout_text
+        assert 'Indirect Member of group: c' in cmd_output.stdout_text
+
+        # clear sssd_cache
+        clear_sssd_cache(master)
+
+        # user lookup
+        # at this point, testuser001 has the following group memberships
+        # Member of groups: a, ipausers
+        # Indirect Member of group: b, c
+        cmd_output = master.run_command(['ipa', 'user-show', username])
+        assert 'a' in cmd_output.stdout_text
+        assert 'ipausers' in cmd_output.stdout_text
+        assert 'b' in cmd_output.stdout_text
+        assert 'c' in cmd_output.stdout_text
+
+        clear_sssd_cache(client)
+
+        try:
+            cmd = 'ssh -i {} -q {}@{} groups'.format('/tmp/user_ssh_priv_key',
+                                                     username, client.hostname)
+            cmd_output = master.run_command(cmd)
+            assert '{}'.format(username) in cmd_output.stdout_text
+            assert 'a' in cmd_output.stdout_text
+            assert 'b' in cmd_output.stdout_text
+            assert 'c' in cmd_output.stdout_text
+        except Exception as e:
+            pytest.fail('Authentication using SSH key not successful', e)
+
+        # add member
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'b', '--users', username])
+        assert 'Group name: b' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member groups: a' in cmd_output.stdout_text
+        assert 'Member of groups: c' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        # now check ssh on the client
+        clear_sssd_cache(client)
+        client.put_file_contents('/tmp/user_ssh_priv_key',
+                                 ssh_key_file_contents)
+        client.run_command('chmod 600 /tmp/user_ssh_priv_key')
```

Comment:
``['chmod', '600', '/tmp/user_ssh_priv_key']``

Reply:
fixed it